### ISSUE TITLE
Add an optional Oracle String Field for orgs

### ISSUE THREAD

**Issue by arakla** [sc0v/binder-app#217]
This way we don't keep files around that have oracle strings, especially if they are old. Additionally, we could have Advisors/Fin Admins update it.

**Comment by pkoenig10** [comment]
This smells of something that Student Activities would have a problem with. Are we allowed to store the Oracle strings for other orgs?

**Comment by arakla** [comment]
We would need work with them on this and it would only be for some orgs (not the Student Activities ones). This way, we can have a report that runs and will include all information needed for the transfers instead of someone sitting down and going through everything in binder to make a spreadsheet for the transfers.

**Comment by pkoenig10** [comment]
Everything else can be done through a properly configured report. If the only thing they have to do is look up Oracle strings for each org, that's not a big deal.

**Comment by sclark** [comment]
I talked with Andrew and we are not allowed to do this in binder.

There may still be something to be said for automatically generating reports for each org. Could you point me toward an example report in the 2016 Carnival drive folder?

**Comment by sclark** [comment]
I talked with Andrew and we are not allowed to do this in binder.

There may still be something to be said for automatically generating reports for each org. @arakla, could you point me toward an example report in the 2016 Carnival drive folder?

**Comment by arakla** [comment]
@sclark
So... Technically speaking, Student Activities can only speak to SA Orgs, unless different policies have been put in place with the creation of SLICE. The reason I say this is that all orcale strings are available to any authenticated user on the CMU finance website. Storing the Oracle strings makes more sense for all non-SA orgs.

Also, and please avoid mentioning this to SA staff, but certain other orgs that do billing the way we do store Oracle strings and verify them with their owners before charging them.

As for the reports,  I can provide an example once I'm at a computer, but I am not referring to org specific ones here, but am taking about the semesterly report/spreadsheet given to Rita (or Tiffany) now to charge all the orgs at once instead of filling out a form for each org, because that is silly.

**Comment by sclark** [comment]
@arakla I did not know this about charging other orgs, but I will ask Zilei to see what functionality she wants going forward.

And okay I think I understand the report now, but an example would be helpful. When you get a chance, could you email it to me?

**Comment by sclark** [comment]
Aamer...you know better. It's andrew.

**Comment by sclark** [comment]
![bitmoji](https://render.bitstrips.com/v2/cpanel/10216848-196926656_11-s4-v1.png?transparent=1&palette=1&width=246)

**Comment by sclark** [comment]
ps...don't be mad at Stephen. You know I always  find everything out and I'm different than the rest of the staff so you're fine :) @arakla

**Comment by sclark** [comment]
@arakla I am sorry! I told Andrew I was talking with you guys from past Carnivals and he snooped around my laptop when I left

**Comment by sclark** [comment]
@arakla I am sorry! I told Andrew that some old members of SCC have some concerns and differing opinions about this years changes to Carnival and he snooped around my laptop when I left

**Comment by arakla** [comment]
@sclark No worries!
So, for Andrew, 2 parts:
1) Student Activities & StuGov technically doesn't/can't set policies for what orgs can store as far as I know; Plus I am not saying to store SA strings, but non-SA strings as they don't change year to year and that way advisors for no SA orgs don't need to email back and forth to ensure the right string is updated. Furthermore, by having them verify the strings in binder, we could set it up as authorization to charge the string by verifying it annually.
2) Smh Andrew, at least make your own account 😝 

And Stephen, I still owe you what I am referring to; I'll try to get that in the next few days; But obviously move-on, etc... takes priority over this

**Comment by sclark** [comment]
Definitely. We will probably have to delay some of these features and bug fixes until at least after move on, but I would love to see those and hopefully help Zilei out by having these reports generated automatically.
